File: Android/ConsumoCredito/app/src/main/java/com/buzzo/consumicredito/Month.kt
package com.buzzo.consumicredito

enum class Month (val month: String) {

    Gen ("01"),
    Feb ("02"),
    Mar ("03"),
    Apr ("04"),
    Mag ("05"),
    Giu ("06"),
    Lug ("07"),
    Ago ("08"),
    Set ("09"),
    Ott ("10"),
    Nov ("11"),
    Dic ("12");

    companion object {
        fun getMonthFromValue(value: String): String {
            var monthName = ""
            for (month in Month.values()) {
                if (month.month == value) {
                    monthName = month.name
                    break
                }
            }
            return monthName
        }
    }

}

File: MACOSX/ConsumoCredito/ConsumoCredito/Utility/Month.swift
//
//  Month.swift
//  ConsumoCredito
//
//  Created by Andrea Turconi on 29/09/2020.
//

import Foundation

class Month {
    
    enum MonthType: String, CaseIterable {
        case Gen = "01"
        case Feb = "02"
        case Mar = "03"
        case Apr = "04"
        case Mag = "05"
        case Giu = "06"
        case Lug = "07"
        case Ago = "08"
        case Set = "09"
        case Ott = "10"
        case Nov = "11"
        case Dic = "12"
    }
    
    public func getMonthFromValue(value: String) -> String {
        var monthName = ""
        for month in MonthType.allCases {
            if month.rawValue == value {
                monthName = String(describing: month)
            }
        }
        return monthName
    }
    
}
